src/caselawclient/models/judgments.py

```diff
@@ -171,19 +171,56 @@ def has_court(self) -> bool:
 
         return True
 
+    # attribute name, value which passes validation, error message
+    VALIDATION_ATTRIBUTES: list[tuple[str, bool, str]] = [
+        (
+            "is_failure",
+            False,
+            "This judgment has failed to parse",
+        ),
+        (
+            "is_held",
+            False,
+            "This judgment is currently on hold",
+        ),
+        (
+            "has_name",
+            True,
+            "This judgment has no name",
+        ),
+        (
+            "has_ncn",
+            True,
+            "This judgment has no neutral citation number",
+        ),
+        (
+            "has_valid_ncn",
+            True,
+            "The neutral citation number of this judgment is not valid",
+        ),
+        (
+            "has_court",
+            True,
+            "This judgment has no court",
+        ),
+    ]
+
     @cached_property
     def is_publishable(self) -> bool:
-        if (
-            self.is_held
-            or not self.has_name
-            or not self.has_ncn
-            or not self.has_valid_ncn
-            or not self.has_court
-        ):
-            return False
+        for function_name, pass_value, _ in self.VALIDATION_ATTRIBUTES:
```

Comment:
Is it worth this just being `return not self.validation_failure_messages()` with an explanatory comment, so the code is not duplicated?